tests/test_transforms/test_decomposition/test_model_based.py

```diff
@@ -337,3 +337,38 @@ def test_stride_transform(forward_stride_datasets, decompose_model, context_size
 
     assert not transformed.df.iloc[context_size:10].isna().any().any()
     assert transformed.df.iloc[10:].isna().all().any()
+
+
+@pytest.mark.parametrize("features_to_ignore", (["target"], ["regressor_1"], ["target", "regressor_1"]))
+def test_repetitive_call_before_pipeline_with_ignore(outliers_solid_tsds, features_to_ignore):
+    """Problem arises when decomposition transform is applied the second time to the same `TSDataset`.
```

Comment:
An empty line should be added here.

I'll approve, and you could do this if you think it is necessary.